tools/startSubmodulesTRAVIS.sh

```diff
@@ -69,10 +69,10 @@ else
 	echo "    HEAD points to a common commit with hash $commitHash"
 fi
 echo "3 - Verifying if last non merge commit is a subproject commit:"
-libModuleHash=$(git show "$commitHash" | grep "+Subproject commit" | cut -f3 -d" ")
-strLength=${#libModuleHash}
-echo "    Subproject commit hash: $libModuleHash"
-if [ \( -n "$libModuleHash" \) -a \( "$strLength" -eq 40 \) ]; then
+libModuleHashes=$(git show "$commitHash" | grep "+Subproject commit" | cut -f3 -d" ")
+strLength=${#libModuleHashes}
+echo "    Subproject commit hash: $libModuleHashes"
+if [ \( -n "$libModuleHashes" \) -a \( "$strLength" -gt 40 \) ]; then
```

Comment:
I think changing `-eq` to `-gt` will break submodule commits that only contain one hash. It should be `-ge` (greater than or equal).